### PR TITLE
Add UI samples to storybook

### PR DESCRIPTION
**Pull request by yoching**
I've added these UI elements to storybook:
- Banner
- Card
- CopyField
- ErrorCard
- FoldToggle
- KeyboardShortcut
- Modal
- Navigation

### CODE REVIEW

File: storybook/stories/Stories/UI/Navigation.elm [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+
+sampleNavItems : Array.Array (N.NavItem Msg)
+sampleNavItems =
+    [ C.ExternalHref "link 1" |> N.navItem "item 1"
+    , C.ExternalHref "link 2" |> N.navItem "item 2"
+    , C.ExternalHref "link 3" |> N.navItem "item 3"
+    , C.ExternalHref "link 4" |> N.navItem "item 4"
+    ]
+        |> Array.fromList
```
Comment (hojberg) [inline storybook/stories/Stories/UI/Navigation.elm:40]: What do we get from Array over List here?

File: storybook/stories/Stories/UI/ui.stories.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+export default { title: "UI" };
+
+import { initElmStory } from "../../initElmStory.js";
+import Icon from "./Icon.elm";
+import Button from "./Button.elm";
+import Card from "./Card.elm";
+import ErrorCard from "./ErrorCard.elm";
+import Banner from "./Banner.elm";
+import CopyField from "./CopyField.elm";
```
Comment (hojberg) [inline storybook/stories/Stories/UI/ui.stories.js:9]: I think this one also has a webcomponent associated with it that we'd want to import in the js world.

**Review by hojberg — COMMENTED** [review]
Glorious!! I add a few comments, but this is so awesome, thank you!

File: storybook/stories/Stories/UI/ui.stories.js (hunk 2 above)
Comment (yoching) [reply storybook/stories/Stories/UI/ui.stories.js:9]: Ok. So are there different ways to use Elm components, other than writing like this?

Comment (hojberg) [reply storybook/stories/Stories/UI/ui.stories.js:9]: Yeah there's a few Elm modules that have a WebComponent pair to do native JavaScript stuff. In this case its `src/UI/CopyOnClick.js`, which need to be included in the bundle for copy to work (the other examples are `OnClickOutside.js`, `EmbedSvg.js`, `EmbedMermaid.js`, and `EmbedKatex.js` FWIW).

**Review by hojberg — APPROVED** [review]

File: storybook/stories/Stories/UI/Navigation.elm (hunk 1 above)
Comment (yoching) [reply storybook/stories/Stories/UI/Navigation.elm]: It was to access one element by its index, but I change it to simply use List.

File: storybook/stories/Stories/UI/ui.stories.js (hunk 2 above)
Comment (yoching) [reply storybook/stories/Stories/UI/ui.stories.js:9]: I see. I changed the code to import CopyOnClick.js

**Comment by hojberg** [comment]
🎉

**Pull request merged**